### PR TITLE
Add a GitHub Actions workflow

### PR DESCRIPTION
**Pull request by athomas**

### CODE REVIEW

**Comment by athomas** [comment]
https://github.com/dart-lang/dartbug.com/pull/18 fixes the test.

**Review by kevmoo — APPROVED** [review]
Want to rebase first to get green before you land?

**Comment by athomas** [comment]
No rush, so I'll merge this tomorrow when GitHub Actions hopefully work again.

**Pull request merged**
